wandb/sdk/artifacts/artifact.py

```diff
@@ -2000,11 +2000,6 @@ def link(self, target_path: str, aliases: Optional[List[str]] = None) -> None:
         Raises:
             ArtifactNotLoggedError: if the artifact has not been logged
         """
-        if not self._ttl_is_inherited:
```

Comment:
Check not needed since `artifact.link` makes calls to `link_artifact`. Don't want duplicate warnings